test/integration/TrueLender2.test.ts

```diff
@@ -86,6 +93,9 @@ describe('TrueLender2', () => {
     await poolFactory.createPool(usdt.address)
     usdtLoanPool = TrueFiPool2__factory.connect(await poolFactory.pool(usdt.address), owner)
 
+    await mockCreditAgency.mock.borrowLimit.withArgs(tusdLoanPool.address, OWNER).returns(parseEth(100_000_000))
+    await mockCreditAgency.mock.borrowLimit.withArgs(usdtLoanPool.address, OWNER).returns(parseEth(100_000_000))
```

Comment:
I'm fine with using mock in tests, but we need to add at least one integration test with the real credit agency

Reply:
By integration I mean a unit test but with a real contract :)

Reply:
Nevermind, didn't notice these are integration tests